### PR TITLE
User interface improvements

### LINKED ISSUE #14: Usability: Meaning of Buttons
Provide context information for buttons. You can find out about most buttons on the landing page, but having to look up functionality there is not a great user experience. Having parts of the explanation crossed out makes it even harder.

Looking at these options, a user should immediately understand what these options do and why they might not be available.

You don’t know that triggering an “Ordinary Scan” will deactivate other options unless it is too late. Though you also wouldn’t find out about that on the about page.

aa1c4cd8-c0d5-46e3-8edf-47028a04b6d6

In this example, I have no idea why I cannot edit any details.

### PR DESCRIPTION
**Pull request by dennis531**
Changes:
- Redirect user to login page after logout
  -  Logout page deleted
  - Show logout message
- Reorganize base template
- Responsive navigation bar
- Separate detail page in two tabs
- More responsiveness

Close #14

### CODE REVIEW

File: src/deterrers-app/hostadmin/templates/host/detail.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        <a href="{% url 'update_host_detail' ipv4=host_ipv4 %}?next={{request.path}}" class="btn btn-primary" title="Edit host details, e.g. internet service profile or host-based firewall tool. The host can not be edited during scans.">
+            Edit Details
+        </a>
+        {% endif %}
+    </div>
+
+    <hr>
+
+    <nav class="nav nav-tabs host-detail-tabs">
+        <a class="nav-link {% if active_tab == "perimeter" %}active{% endif %}" href="{% url 'host_detail' ipv4=host_ipv4 tab='perimeter' %}">Perimeter Firewall</a>
```
Comment (dennis531) [inline src/deterrers-app/hostadmin/templates/host/detail.html:43]: I am not sure if we should name this tab differently (e.g. "Actions", "General")?

Comment (nwintering) [reply src/deterrers-app/hostadmin/templates/host/detail.html:43]: Yes, I will think about a fitting name and change it. "General" is probably the best choice...

**Review by nwintering — COMMENTED** [review]
Looks much better and solves most of issue #14

**Comment by nwintering** [comment]
For further improvement, we could even transfer most of the "details"-table into the General-tab. Only the host-based firewall field would go into the "Host-based Firewall"-tab.
This would also mean that the selection of Internet Service Profile and Host-based Firewall program should be separated into different forms.

**Pull request merged**